js/packages/phoenix-evals/package.json

```diff
@@ -56,6 +56,7 @@
     "@arizeai/phoenix-client": "workspace:*",
     "@types/mustache": "^4.2.6",
     "@types/node": "^24.0.10",
+    "msw": "^2.11.6",
```

Comment:
not ideal but trying not to move to beta https://github.com/vercel/ai/issues/8917